### ISSUE TITLE
Miscellaneous tag fixes

### ISSUE THREAD

**Issue by macks2008** [HexxyCraft/modpack#176]
I've noticed in the course of play that there are some things that don't feel like they are tagged correctly, such as sliced fish not counting as fish for the purpose of feeding cats, or this one [I discovered last night](https://discord.com/channels/936370934292549712/1099785532118143127/1160476603168600085) where stripped edified logs cannot be used to make Create's (andesite/brass/copper) casings because they are not associated with this tag.
![image](https://github.com/HexxyCraft/modpack/assets/7110033/d5b1d2aa-fbe6-4482-9692-b55ce7afef66)
I also might want to expand glimmering flowers' tags and uses, because goodness there are a lot of those around our current map! (*Cough* @Buggy123 *cough*)… But I think that would make more sense as a separate enhancement issue since the scope I have in mind is a bit more broad.

Let me know your thoughts below, of course. And as before, I'd like to take a crack at this one myself if you'd like to assign it to me. I've done at least some work with tags in the past, and the stuff I'm asking *seems* simple enough (*knocks on wood* "Murphy, don't you dare!")

**Comment by macks2008** [comment]
One question I had from last time, which may or may not be relevant to the stripped edified log tagging issue: What's the purpose of the script you have that changes the tagging of planks??

**Comment by object-Object** [comment]
[this one](https://github.com/HexxyCraft/modpack/blob/master/scripts/recipes/chests.zs)? it doesn't modify any existing tags, the purpose is to disable using edified planks to craft chests so the edified panel recipe works

**Comment by object-Object** [comment]
feel free to have at it though, PRs are always welcome.

**Comment by Twisted-Code** [comment]
Yes that one, and that's what I thought its purpose might be. In that case I think I would like to discuss Petra's choice of recipe for the panels, but I'll do that on her tracker. (As far as this pack is concerned, I would've just added a disambiguating recipe for the panels, it's probably simpler)
Edit 2025-01-22: I totally forgot about this issue didn't I. I guess the pack is obsolete anyway but… Whoops
